scripts/build-alloy.js

```diff
@@ -64,7 +64,7 @@ const argv = yargs(hideBin(process.argv))
   })
   .option("minify", {
     type: "boolean",
-    default: false,
+    default: true,
```

Comment:
Changed the CLI minify flag to true by default.

Reply:
I was thinking that we'd rename the flag to "dev" which would default to false. So to get the un-minified version you would do "npx @adobe/alloy --dev"